### DISCUSSION TITLE
url formating

### DISCUSSION THREAD

**Discussion by goshawk1980** [givanz/Vvveb#190] [General]
Is it possible to make the urls work without appending the page id?
I would like the urls to be mydomain.com/about-us and not mydomain.com/abous-us-15 :(

The number seems to append automatically when saving the pages/posts

This also affects browsing the site
mydomain.com/anything-15 will also load the about us page :( ... there is no check on the slug, only the id
This could be an issue as it could be considered as duplicate content

**Comment by givanz** [comment]
The id's were added recently to avoid adding page redirects or have broken urls when a page slug changes.

You can revert to the previous format where only slug is used by changing 

```php
'/{slug}-#post_id#' => ['module' => 'content/post/index', 'edit'=>'?module=content/post&post_id={post_id}&type=post'],
'/page/{slug}-#post_id#' => ['module' => 'content/page/index', 'edit'=>'?module=content/post&post_id={post_id}&type=page'],
```
to 

```php
'/{slug}'        => ['module' => 'content/post/index', 'edit'=>'?module=content/post&slug={slug}&type=post'],
'/page/{slug}'   => ['module' => 'content/page/index', 'edit'=>'?module=content/post&slug={slug}&type=page'],
```
in config/routes.php https://github.com/givanz/Vvveb/blob/master/config/routes.php#L145-L148
